CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Linkage Checker Enforcer Rule Change Log
 
+## 1.4.3
+* Made ClassPathEntry.getArtifact() public for the Linkage Checker Gradle plugin
```

Comment:
Fair warning: nothing about ClassPathEntry, including its existence, should be considered stable at this point.